altostra/manifest.json

```diff
@@ -0,0 +1,36 @@
+{
+  "display_name": "Altostra",
+  "maintainer": "support@altostra.com",
+  "manifest_version": "1.0.0",
+  "name": "altostra",
+  "metric_prefix": "altostra.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "Auto instrument cloud resources to send logs and metrics to Datadog.",
+  "guid": "c5b325e5-a55a-4ff2-9c91-3aac9463c6be",
+  "support": "contrib",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Datadog-Altostra Integration",
+  "categories": [
+    "automation",
+    "aws",
+    "cloud",
+    "configuration & deployment",
+    "log collection"
+  ],
+  "type": "check",
+  "is_public": true,
+  "integration_id": "altostra",
+  "assets": {
+    "dashboards": {},
```

Comment:
Need a direct link to the dashboard path provided.

Reply:
@scseanchow We added a dashboard and linked it in the `manifest.json`.